webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -1824,6 +1823,207 @@ class OntologyV2R2RSpec extends R2RSpec {
+      // update label and comment of a property
+      val ontoIri            = SharedOntologyTestDataADM.ANYTHING_ONTOLOGY_IRI_LocalHost
+      val propertyIri        = ontoIri + "#hasOtherNothing"
+      val propertyValueIri   = ontoIri + "#hasOtherNothingValue"
+      val propertyIriEncoded = URLEncoder.encode(propertyIri, "UTF-8")
+
+      Delete(
+        s"/v2/ontologies/properties/comment/$propertyIriEncoded?lastModificationDate=$anythingLastModDate"
+      ) ~> addCredentials(
+        BasicHttpCredentials(anythingUsername, password)
+      ) ~> ontologiesPath ~> check {
+        val responseStr = responseAs[String]
+
+        assert(status == StatusCodes.OK, response.toString)
+        val responseJsonDoc    = JsonLDUtil.parseJsonLD(responseStr)
+        val updatedOntologyIri = responseJsonDoc.body.value("@id").asInstanceOf[JsonLDString].value
+        assert(updatedOntologyIri == ontoIri)
+        val graph               = responseJsonDoc.body.requireArray("@graph").value
+        val property            = graph.head.asInstanceOf[JsonLDObject]
+        val returnedPropertyIri = property.requireString("@id")
+        returnedPropertyIri should equal(propertyIri)
+        assert(property.value.get(OntologyConstants.Rdfs.Comment) == None)
+        val lastModDate = responseJsonDoc.body.requireDatatypeValueInObject(
+          key = OntologyConstants.KnoraApiV2Complex.LastModificationDate,
+          expectedDatatype = OntologyConstants.Xsd.DateTimeStamp.toSmartIri,
+          validationFun = stringFormatter.xsdDateTimeStampToInstant
+        )
+
+        assert(lastModDate.isAfter(fooLastModDate))
+        anythingLastModDate = lastModDate
+
+        // load back the ontology to verify that the updated property still is editable
+        val encodedIri = URLEncoder.encode(s"${SharedOntologyTestDataADM.ANYTHING_ONTOLOGY_IRI_LocalHost}", "UTF-8")
+        Get(
+          s"/v2/ontologies/allentities/${encodedIri}"
+        ) ~> ontologiesPath ~> check {
+          val responseStr: String = responseAs[String]
+          assert(status == StatusCodes.OK, response.toString)
+          val responseJsonDoc = JsonLDUtil.parseJsonLD(responseStr)
+
+          val graph           = responseJsonDoc.body.requireArray("@graph").value.map(_.asInstanceOf[JsonLDObject])
+          val nothingValue    = graph.filter(_.requireString("@id") == propertyValueIri).head
+          val isEditableMaybe = nothingValue.maybeBoolean(OntologyConstants.KnoraApiV2Complex.IsEditable)
+          isEditableMaybe should equal(Some(true))
+        }
+      }
+    }
+
+    "update the property guiElement and check if the property is still correctly marked as `isEditable: true`" in {
+      // update label and comment of a property
+      val ontoIri          = SharedOntologyTestDataADM.ANYTHING_ONTOLOGY_IRI_LocalHost
+      val propertyIri      = ontoIri + "#hasOtherNothing"
+      val propertyValueIri = ontoIri + "#hasOtherNothingValue"
```

Comment:
Cannot these 3 be defined once in the entire file and reused instead of dining them in every test?